features/src/main/feature/camel-features.xml

```diff
@@ -427,6 +427,12 @@
         <bundle dependency='true'>wrap:mvn:software.amazon.awssdk/sqs/${aws-java-sdk2-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-aws2-sqs/${project.version}</bundle>
     </feature>
+    <feature name='camel-aws2-step-functions' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version="${aws-java-sdk2-version}">awssdk</feature>
```

Comment:
I think this is ok, just to mention, AWS SDK versions are
2.24.0 in dependency tree
2.25.15 in camel-karaf

Reply:
did not switch to it, was looking at 4.4.x, you are right, the versions correspond. Please disregard previous comment